### DISCUSSION TITLE
Filter like integer using contains

### DISCUSSION THREAD

**Discussion by jorgeleal12** [prisma/prisma#9047] [Q&A]
filter like integer
using contains presents error

**Comment by ryands17** [comment]
@jorgeleal12 👋 

You cannot filter an integer using contains. If you need a use case where contains is needed, then it's best to use a string and not integer.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
